dist/src/helpers/RoomHelper.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
```

Comment:
Ik zie dat deze files ondanks de toevoeging aan de `.gitignore` file toch in version control zijn gekomen. Je zal ze expliciet moeten verwijderen met `git rm` om ze er weer uit te krijgen.